sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/utils/ResourceManagerUtils.java

```diff
@@ -249,6 +255,15 @@ public static class InternalRuntimeContext {
         private static DelayProvider delayProvider = new ResourceDelayProvider();
         private static Scheduler reactorScheduler = Schedulers.parallel();
 
+        /**
+         * Creates a new instance of InternalRuntimeContext.
+         *
+         * @deprecated InternalRuntimeContext is a static inner class.
+         */
+        @Deprecated
```

Comment:
Not Deprecated

Reply:
Fixed in the [commint#7642a91](https://github.com/Azure/azure-sdk-for-java/pull/43192/commits/7642a91a0622459f71ea4750e77e3de93c23746b)